static/releases.json

```diff
@@ -1,62 +1,140 @@
-[
-  {
-    "branch": "1.16.x",
-    "initialDate": "2025-11-10",
-    "ossPolicyEnd": "2026-12-31",
-    "commercialPolicyEnd": "2027-12-31"
-  },
-  {
-    "branch": "1.15.x",
-    "initialDate": "2025-05-12",
-    "ossPolicyEnd": "2026-06-30",
-    "commercialPolicyEnd": "2032-06-30"
-  },
-  {
-    "branch": "1.14.x",
-    "initialDate": "2024-11-11",
-    "ossPolicyEnd": "2025-12-31",
-    "commercialPolicyEnd": "2026-12-31"
-  },
-  {
-    "branch": "1.13.x",
-    "initialDate": "2024-05-13",
-    "ossPolicyEnd": "2025-06-30",
-    "commercialPolicyEnd": "2026-06-30"
-  },
-  {
-    "branch": "1.12.x",
-    "initialDate": "2023-11-13",
-    "ossPolicyEnd": "2024-12-31",
-    "commercialPolicyEnd": "2025-12-31"
-  },
-  {
-    "branch": "1.11.x",
-    "initialDate": "2023-05-09",
-    "ossPolicyEnd": "2024-06-30",
-    "commercialPolicyEnd": "2025-06-30"
-  },
-  {
-    "branch": "1.10.x",
-    "initialDate": "2022-11-07",
-    "ossPolicyEnd": "2023-12-31",
-    "commercialPolicyEnd": "2024-12-31"
-  },
-  {
-    "branch": "1.9.x",
-    "initialDate": "2022-05-12",
-    "ossPolicyEnd": "2023-06-30",
-    "commercialPolicyEnd": "2029-06-30"
-  },
-  {
-    "branch": "1.8.x",
-    "initialDate": "2021-11-11",
-    "ossPolicyEnd": "2022-11-24",
-    "commercialPolicyEnd": "2024-02-24"
-  },
-  {
-    "branch": "1.7.x",
-    "initialDate": "2021-05-12",
-    "ossPolicyEnd": "2022-05-19",
-    "commercialPolicyEnd": "2023-08-24"
-  }
-]
+{
+  "micrometer": [
+    {
+      "branch": "1.17.x",
+      "initialDate": "2026-05-12",
+      "ossPolicyEnd": "2027-06-30",
+      "commercialPolicyEnd": "2028-06-30"
+    },
+    {
+      "branch": "1.16.x",
+      "initialDate": "2025-11-10",
```

Comment:
Not related to the changes here, but I noticed this date is wrong since we released a bit earlier.